outlook/src/classes/Company.ts

```diff
@@ -175,7 +175,12 @@ class Company {
         return this.id == ID_COMPANY_EMPTY;
     }
 
-
+    /***
+     * Returns True if the company has IAP insights
+     */
+    hasInsights(): boolean {
```

Comment:
I find it strange that we condition the whole display of the "getNoCompanyInsightsSection" (that basically says "sorry, no info") and "getCompanyInsightsSection" based on the fact that the company was enriched or not.

I mean there could be valuable information manually input on the company, like the location, the phone, the logo, we don't especially need IAP for those, the user could have filled in his database manually.

Not showing anything when a record is not enriched seems a bit harsh.
People may want to work with the addon without paying for this extra service, and I feel like the addon should behave properly in that case too.

Maybe something we need to discuss with Luc? Or did I miss something?

Reply:
This was discussed with Luc, and it has been decided to only check if a contact has Iap insights to determine if it can be enriched or not.

However I think that we can improve this, by displaying the contact infos that we already have and put the enrich button in the bottom if the contact has not been enriched before, I think this is much better then directly showing an empty section even if we have some infos.

Can you tell me what do you think about it? this should also be discussed with Luc I think?

Reply:
Indeed, I think this can be improved to show the information we have on the company no matter if the company has been enriched or not (like the address, phone, ...).

This makes sense to me so I would go ahead and do it, we will see what Luc thinks about it when testing.

Reply:
Okay then, we now always display the info + the button in the bottom if the company can be enriched.